File: connectors/connectors-rabbitmq/src/main/scala/com/chaojianok/flink/connectors/rabbitmq/scala/utils/PropertyUtil.scala
package com.chaojianok.flink.connectors.rabbitmq.scala.utils

import java.io.FileInputStream
import java.util.Properties

object PropertyUtil {

  def loadProperties(): Properties = {
    val properties = new Properties()
    val path = Thread.currentThread().getContextClassLoader.getResource("application.properties").getPath
    properties.load(new FileInputStream(path))
    return properties
  }

  def getValue(key: String): String = {
    return loadProperties.getProperty(key)
  }

}

File: connectors/connectors-rabbitmq/src/main/java/com/chaojianok/flink/connectors/rabbitmq/java/utils/PropertyUtil.java
package com.chaojianok.flink.connectors.rabbitmq.java.utils;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.io.IOException;
import java.io.InputStream;
import java.util.Properties;

public class PropertyUtil {

    private static Logger logger = LoggerFactory.getLogger(PropertyUtil.class);

    public static Properties loadProperties() {
        Properties properties = new Properties();
        InputStream in = PropertyUtil.class.getClassLoader().getResourceAsStream("application.properties");
        try {
            properties.load(in);
        } catch (IOException e) {
            logger.warn("加载配置文件失败，warn=" + e);
        }
        return properties;
    }

    public static String getValue(String key) {
        return loadProperties().getProperty(key);
    }

}
